moveit_core/robot_state/src/conversions.cpp

```diff
@@ -39,24 +39,24 @@
 #include <geometric_shapes/shape_operations.h>
 #include <tf2_eigen/tf2_eigen.h>
 #include <boost/lexical_cast.hpp>
+#include <moveit/robot_state/log.h>
```

Comment:
Uses logger created before. In case it's desired to create the logger here instead, please say it.

Reply:
I agree with @henningkayser here. Adding a log.h each time we want a logger is overkill. see comment here: https://github.com/ros-planning/moveit2/pull/40#discussion_r264481130